src/client/utils/collections.js

```diff
@@ -0,0 +1,74 @@
+import { useQuery } from 'react-query';
+import collectionPlaceholderPng from '../resources/icon.png';
+import { getCollection, getCollections } from '../../api/collection';
+
+const collectionQueryConfig = {
+  staleTime: 1000 * 60 * 60, // time until data in cache considered stale if cache not invalidated
+  cacheTime: 1000 * 60 * 60, // time before cache labeled as inactive to be garbage collected
+};
+
+// fallback collection
+const loadingCollection = {
+  name: 'Loading...',
+  description: 'loading...',
+  loadingCollection: true,
+  comments: [],
+  subitems: [],
+  voteScore: 0,
+  views: 0,
+  image: {
+    backgroundUrl: collectionPlaceholderPng,
```

Comment:
I wonder if it would not be better to have a standard placeholder instead of the graasp icon? I think the blue icon vs the grey skeleton clash a bit.

Reply:
mmmm What do you think we should put instead?

Reply:
https://material-ui.com/components/skeleton/#Facebook.js Maybe just a rectangle? There's an example using cards.